### ISSUE TITLE
Refresh metadata doesn't work

### ISSUE THREAD

**Issue by siegfried2p** [gboudreau/XBMCnfoMoviesImporter.bundle#101]
when i hit refresh metadata nothing happens
but if i hit match..  it displays correct metadata and i can apply that
it's sort of working but not full automation especially with huge library
can it be fixed? or is it because of plex it self?
[com.plexapp.agents.xbmcnfo.log](https://github.com/gboudreau/XBMCnfoMoviesImporter.bundle/files/1778193/com.plexapp.agents.xbmcnfo.log)

**Comment by SlrG** [comment]
Sadly this is a know problem. Plex caches a lot of things and does not refresh them, if the movie itself seems unchanged. So there are only workarounds, which are a pain either way.

1.) You can delete and recreate the whole library.
2.) Move the movie to another dir, refresh, move it back and refresh again.
3.) Do a rematch like you already found yourself.

**Comment by siegfried2p** [comment]
thanks, wish plex will fix it :)

**Comment by jwaltrip4** [comment]
I know this is about 9 months old.  Anything new on this...
Just want to make sure I know issue....

I just set up a new server (migrated off of old.  Scanned in movies, but not sure I did right with your importer.

Got Importer properly set up,  did another rescan meta data...  and nothing is updating.

Have tried changing tags (like I have done in past) and rescan.  No updates in plex.

Fix you say is to delete lib from plex and re-create?  this will scan fresh and pick up info in NFO and folder?
